AnkiDroid/src/main/java/com/ichi2/libanki/CollectionV16.kt

```diff
@@ -156,6 +156,27 @@ class CollectionV16(
         return cardIdsList
     }
 
+    override fun findNotes(
+        query: String?,
+        order: SortOrder,
+    ): List<Long> {
+        val adjustedOrder = if (order is SortOrder.UseCollectionOrdering) {
+            @Suppress("DEPRECATION")
+            SortOrder.BuiltinSortKind(
+                get_config("noteSortType", null as String?) ?: "noteFld",
+                get_config("sortBackwards", false) ?: false,
```

Comment:
Sorry, I should have noticed this last time:
```suggestion
                get_config("browserNoteSortBackwards", false) ?: false,
```